requirements.txt

```diff
@@ -14,7 +14,7 @@ docker >= 4.0
 fastapi >= 0.70
 fsspec >= 2022.5.0
 griffe >= 0.20.0
-httpx[http2] >= 0.23
+httpx[http2] == 0.23.1  # https://github.com/PrefectHQ/prefect/pull/8004#issuecomment-1369457371
```

Comment:
We should almost never pin versions — you can exclude a version e.g. `!= 0.23.2` if needed or add an upper bound. I think casting to strings is the correct solution here though.